teuthology/suite/merge.py

```diff
@@ -14,7 +14,7 @@
 TEUTHOLOGY_TEMPLATE = MappingProxyType({
```

Comment:
~~I guess this also needs to be converted into `dict` type because we removed support for `MappingProxyType` from `deep_merge`?~~

Just noticed you converted that into `dict` in line 132 so we should be good then!